### PR TITLE
Develop basic structure for onelive landing page

### PR DESCRIPTION
**Pull request by Gravewalker666**
## Purpose
Develop the basic structure for oneLive landing page

## Goals
 Implementing the new UI according to the design

## Approach
 
screencapture-localhost-SEF-sef-site-src-main-webapp-onelive-2019-12-10-05_18_07

## Security checks
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

## Related PRs
 N/A

## Test environment
Chrome, Apache2, openJdk1.8, Ubuntu 19.04
 
## Learning
N/A

### CODE REVIEW

**Comment by jayasanka-sack** [comment]
- [x] Make "Onelive is an interactive ... " text bigger.
- [x] Replace all dummy texts. Refer the site doc.
- [x] Link see upcoming events with our facebook page's event page
- [ ] Add a mobile preview.
- [x] Add a preview link.
   how? 🤔
   - go to repo settings > options > git hub pages
   -  set source to master
   -  visit http://gravewalker666.github.io/sef-site/src/main/webapp/ and you will see the magic!
   - enjoy!
- [x]  Remove > character in PR description. Hard to read gray texts. Lol 😄

**Comment by jayasanka-sack** [comment]
Make "Be a onelive speaker" and "Recommend a speaker" sections equal in size.

**Comment by Gravewalker666** [comment]
## Preview
https://gravewalker666.github.io/sef-site/src/main/webapp/onelive

- There is a small issue in FAQ component. I will solve it and put it in the component page so that everyone can use it hereafter.

**Comment by jayasanka-sack** [comment]
@Gravewalker666 I'm attaching some screenshots taken from my browser,

- [x] Hero image is bigger than the viewport
- [x] "Be a speaker" sections' image isn't rendered properly
- [ ] Let's make "be a speaker" section &  "Recommend a speaker" sections equal in size

image

image

**Review by Piumal1999 — APPROVED** [review]

File: src/main/webapp/onelive.html [hunk 1]
```diff
@@ -28,6 +28,230 @@
...
 <main>
     <!--main content goes here-->
+    <section class="section">
+        <div class="container">
+            <div class="container mt-md-5 d-none d-md-block" style="position: absolute; z-index: 1000">
+                <h1 class="mt-md-5 mb-md-5" style="font-size: 3.3rem; line-height: 1.5;"><span class="font-weight-700">Meet the</span>
+                    Real Life Heroes</h1>
+                <h1 class="mb-0 text-darker font-weight-lighter text-uppercase">one <span
+                        class="font-weight-bold">Live</span></h1>
+                <p class="text-darker">Lorem ipsum dolor sit amet</p>
```
Comment (Piumal1999) [inline src/main/webapp/onelive.html]: Remove dummy text

File: src/main/webapp/onelive.html [hunk 2]
```diff
@@ -49,11 +273,82 @@
...
+    });
+
+    //function to load profile data
+    function loadProfiles() {
+        $.ajax({
+            type: 'get',
+            url: 'assets/content/json/v3.0.0/onelive.json',
+            dataType: 'json',
+            success: function (data) {
+
```
Comment (Piumal1999) [inline src/main/webapp/onelive.html:322]: Use the updated function which has animated loading (check https://github.com/sef-global/sef-site/blob/master/src/main/webapp/examples/team.html )

File: src/main/webapp/onelive.html (hunk 1 above)
Comment (Gravewalker666) [reply src/main/webapp/onelive.html]: I commented it so if someone needs to put a slogan there we can uncomment it and replace the dummy text. (There was a space to put a slogan in the design but none provided that why I kept it.)

File: src/main/webapp/onelive.html (hunk 2 above)
Comment (Gravewalker666) [reply src/main/webapp/onelive.html:322]: Done!

**Comment by Piumal1999** [comment]
Well Done @Gravewalker666

**Pull request merged**